app/renderer/lib/client-frameworks/java.js

```diff
@@ -208,6 +211,57 @@ ${this.indent(code, 4)}
     return `driver.setClipboardText("${clipboardText}");`;
   }
 
+  // File Transfer
+
+
+  codeFor_pushFile (varNameIgnore, varIndexIgnore, pathToInstallTo, fileContentString) {
+    return `driver.pushFile("${pathToInstallTo}", ${fileContentString});`;
+  }
+
+  codeFor_pullFile (varNameIgnore, varIndexIgnore, pathToPullFrom) {
+    return `byte[] fileBase64 = driver.pullFile("${pathToPullFrom}");`;
+  }
+
+  codeFor_pullFolder (varNameIgnore, varIndexIgnore, folderToPullFrom) {
+    return `byte[] fileBase64 = driver.pullFolder("${folderToPullFrom}");`;
+  }
+
+  // Device Interaction
+
+  codeFor_shake () {
+    return `driver.shake();`;
+  }
+
+  codeFor_lock (varNameIgnore, varIndexIgnore, seconds) {
+    return `driver.lockDevice(${seconds});`;
+  }
+
+  codeFor_unlock () {
+    return `driver.unlockDevice();`;
+  }
+
+  codeFor_isLocked () {
+    return `boolean isLocked = driver.isDeviceLocked();`;
+  }
+
+  codeFor_rotateDevice (varNameIgnore, varIndexIgnore, x, y, radius, rotation, touchCount, duration) {
+    return `driver.rotate(new DeviceRotation(${x}, ${y}, ${radius}, ${rotation}, ${touchCount}, ${duration}));`;
```

Comment:
this may probably be confusing because we only support the limited set of arguments for this API. Random args won't work

Reply:
Reading into this, it seems that the parameters coming from Inspector are for the deprecated `mobileRotation` command, instead of Selenium's rotation endpoints. I'd propose leaving this here for now, so I can later adjust it everywhere.